Q3DC/Q3DC.py

```diff
@@ -1660,22 +1668,20 @@ def test_CalculateDisplacement1(self):
             markupsNode1.GetNthControlPointLabel(1)
         )
 
-        diffXAxis, diffYAxis, diffZAxis, threeDDistance = logic.computeDistance(*args)
-        if diffXAxis != -2.687 or diffYAxis != -10.526 or diffZAxis != -57.452 or threeDDistance != 58.47:
-            return False
-        return True
+        dx, dy, dz, dist = logic.computeDistance(*args)
+        assert (dx, dy, dz, dist) == (-2.687, -10.526, -57.452, 58.47)
```

Comment:
Would it make sense to use https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_almost_equal.html ?

Reply:
Since the results of `computeDistance` are rounded, I'm not sure it really matters. Also, since the return is a tuple, not an ndarray, there would need to be extra code to convert the result and expected values to ndarray. I think the rounding + tuple comparison is sufficient.

Reply:
> I think the rounding + tuple comparison is sufficient.

Makes sense.